summary.Rmd

```diff
@@ -0,0 +1,40 @@
+Summary
+=======
+
+1. Use available tools
+    a. Use ready-made images (e.g. from Docker Hub)
+    b. Dockerfile generators (e.g. repo2docker)
+    c. Project templates (e.g. [cookie cutter](https://github.com/cookiecutter/cookiecutter))
+    d. Linters
+2. Use versioned images
+3. Format for clarity
```

Comment:
Do you also suggest to rename the rule?

Reply:
I would recommend doing so. This is more succinct and doesn't really sacrifice any meaning.

Reply:
https://github.com/nuest/ten-simple-rules-dockerfiles/commit/03e5d657c64983ac0042b79c894c58b69eb957c7